File: goblin/goblin.go
package main

import (
	"fmt"
	"math"
	"math/rand"
	"time"
)

func domath(number float64) {
	var d = math.Pow(number, 15)
	var id = math.Pow(d, 15)
	id++
	fmt.Println(d, id)
	
}
func main() {
	start := time.Now()
	rand.Seed(time.Now().UnixNano())
	var n int
	for i := 0; i < 5000000; i++ {
		n = rand.Intn(10000000)
		f := float64(n)
		domath(f)
	}
	elapsed := time.Since(start)
	fmt.Println(elapsed)

}

File: goblin/goblin.py
from random import randint
from time import time
def domath(number):
    d = pow(number, 5)
    i = pow(d, 5)
    print(d,i)
def main():
    r = time()
    i = 0
    while(i < 5000000):
        n = randint(1, 10000000)
        domath(n)
        i += 1
    ert = time()
    est = ert - r
    print(est, 's')
if __name__ == "__main__":
    main()
